src/ui/components/util/PlayerContext.ts

```diff
@@ -22,6 +23,11 @@ export interface UiContext {
    * Whether a linear ad is currently in progress.
    */
   readonly adInProgress: boolean;
+
+  /**
+   * The selected localization.
+   */
+  readonly localization: AllLocalization;
```

Comment:
[Localization is "the process of adapting a software user interface to a specific culture"](https://developer.mozilla.org/en-US/docs/Glossary/Localization). It's the *act* of translating, not the *result* of it.

Consider renaming the API to "locale" or "language" or something.
```suggestion
  readonly locale: Locale;
```